chain/events/filter/index.go

```diff
@@ -56,27 +61,203 @@ var ddls = []string{
     	version UINT64 NOT NULL UNIQUE
 	)`,
 
-	// version 1.
 	`INSERT OR IGNORE INTO _meta (version) VALUES (1)`,
+	`INSERT OR IGNORE INTO _meta (version) VALUES (2)`,
```

Comment:
Shouldn't we get rid of the previous line?

Reply:
We could but I kept it so we have an explicit trace of all versions that we have used, but yes it is not necessary.

Reply:
Fair enough.